marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/entitySearch/exportSearch.api

```diff
@@ -1,5 +1,5 @@
 {
-    "functionName": "getOpticPlan",
+    "functionName": "exportSearch",
```

Comment:
I think we really need a description for this - based on the code, I'm thinking: "Evaluates the given search query against the given TDE view, returning the results as CSV"